ml/src/main/scala/ai/h2o/sparkling/ml/params/H2OCommonParams.scala

```diff
@@ -62,12 +57,12 @@ trait H2OCommonParams extends H2OMOJOAlgoSharedParams {
 
   def getSplitRatio(): Double = $(splitRatio)
 
-  def getSeed(): Long = $(seed)
-
-  def getNfolds(): Int = $(nfolds)
-
   def getColumnsToCategorical(): Array[String] = $(columnsToCategorical)
 
+  // To be overridden
+  def getSeed(): Long
```

Comment:
qq: It seems like `getSeed` is already generated ( Removed from H2OAlgoCommonParams etc). Should this be removed?

Reply:
Some tests expect this method on parent classes (e.g. https://github.com/h2oai/sparkling-water/blob/master/ml/src/test/scala/ai/h2o/sparkling/ml/models/H2OSupervisedMOJOModelTestSuite.scala#L129). So i defined just prototype of getters and setters in the parent and an implementation is in a generated class.

Another solution could be to duplicate the test for each specific algorithm tested  like this.

I'm not sure which approach is better. I'm ok with both.

Reply:
I see!

What about different approach and make sure tests depends on generated sources via Gradle?

Reply:
>What about different approach and make sure tests depends on generated sources via Gradle?

Can share more about this approach? Tests can utilize generated classes already, but are shared by multiple algorithms.

Reply:
Sorry, my bad. Now I see. What about using reflections in tests?

Reply:
yes, that's possible. Will change it.